components/transferForm.tsx

```diff
@@ -993,14 +1042,100 @@ function TransferForm(
                                       <ErrorMessage
                                         name={`transfers.${index}.values.${value.field}`}
                                       />
+                                      {!hasTzip27SupportWithPoEChallenge(
+                                        props.contract.version
+                                      ) &&
+                                        renderWarning(
+                                          "This version doesn't support message signing, and this functionality will be removed following submission."
```

Comment:
I don't get what you mean. 
> This version doesn't support message signing, and this functionality will be removed
- in the next version?
- in the following version?

Reply:
It happens when:
1. user is on wallet 0.3.4 and creates a proposal for message signing
2. user switch wallet to 0.1.1 on sidebar. 
So, for wallet 0.1.1, it doesn't support message signing.